services/callback/src/main/resources/bootstrap-cloud.yaml

```diff
@@ -14,7 +14,6 @@ spring:
         enabled: true
         backend: ${VAULT_BACKEND}
         profile-separator: '/'
-        application-name: 'callback'
         default-context: ''
         profiles: cloud
```

Comment:
would this profile then be needed or conflict with the other settings?

Reply:
Normally we have in the deploymentConfig something like ACTIVE_PROFILES: cloud, connect-efgs. Removing this 'callback' decouples the vault secret engine from the cloud profile. However it is still the same vault. Therefore I think that the profile here needs to stay.

Reply:
I think is needed and would not conflict with the other settings. This is actually what we are trying to verify, if we have the application name and profiles used  in the federation specific bootstrap yaml the vault error still persists.